packages/dev/core/src/Materials/Node/Blocks/Dual/sceneDepthBlock.ts

```diff
@@ -191,17 +193,28 @@ export class SceneDepthBlock extends NodeMaterialBlock {
             if (state.target === NodeMaterialBlockTargets.Fragment) {
                 return;
             }
+            const textureReadFunc =
+                state.shaderLanguage === ShaderLanguage.GLSL
+                    ? `texture2D(${this._samplerName},`
+                    : `textureSampleLevel(${this._samplerName}, ${this._samplerName + Constants.AUTOSAMPLERSUFFIX},`;

```

Comment:
Same question as above (I guess the answer is the same!).

Reply:
you guess right :D